test/morse_translator_test.rb

```diff
@@ -0,0 +1,61 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/morse_translator'
+
+class MorseTranslatorTest < Minitest::Test
```

Comment:
This tests are awesome! I'm especially impressed that you have tested your helper methods!